diagnostic.md

```diff
@@ -57,6 +74,7 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
+$("body").hide()
```

Comment:
This works but we wanted you to start from an `li` node and work your way back up, like so:

`$('li').parent().parent().hide()`